src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -74,9 +74,9 @@ public async Task<IActionResult> GetEmissionsDataForLocationsByTime([FromQuery]
     [ProducesResponseType(StatusCodes.Status400BadRequest, Type = typeof(ValidationProblemDetails))]
     [HttpGet("bylocation")]
     public async Task<IActionResult> GetEmissionsDataForLocationByTime(
-        [FromQuery, SwaggerParameter(Required = true)] string location, 
-        DateTimeOffset? startTime = null, 
-        DateTimeOffset? endTime = null)
+        [FromQuery, SwaggerParameter(Required = true)] string location,
```

Comment:
I think the queryParams defined in the controller are `startTime` and `endTime`

Reply:
Copied it over from https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/src/CarbonAware.WebApi/src/Models/Parameters.EmissionsDataForLocationsDTO.cs#L16 where it's `time`, `toTime`. Note that we have other ones https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/src/CarbonAware.WebApi/src/Models/Parameters.EmissionsForecastCurrentDTO.cs where the query params are `dataStartAt`, `dataEndAt`. I don't think that we have `startTime ` and `endTime` anywhere in WebAPI though we should probably make them consistent.

Reply:
aah, got it! sorry I got confused since we use startTime, endTime in this endpoint - https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs#L68. 
I agree, we should make it consistent, but that isnt part of this story, so you can ignore for now.